### DISCUSSION TITLE
CLI - Redeploy a specific deployment after setting an environment variable

### DISCUSSION THREAD

**Discussion by jon-ressio** [vercel/community#1616] [Help] [answered]
### Summary

I'm using the CLI to create/override an environment variable for a specific git branch/deployment, but I don't see any way of redeploying that specific deployment (which comes from that git branch via integration) to pick up this environment variable change.

Is there something I'm missing or is that just not supported via the CLI yet?

**Comment by EndangeredMassa** [comment, marked as answer]
Hello @jon-ressio!

You cannot currently trigger a redeploy from the CLI. That is something we'd like to add in the future.

You would have to check out the proper branch/commit first, then do a `vc deploy`.
